data_storage/export_format.md

```diff
@@ -1,14 +1,16 @@
 Export Format
 -------------
 
-- ![](/badge/req.png) A Toxcore data file saved in the root of the client's
-  working directory, with the file name `<profile_name>.tox`
+- ![](/badge/req.png) A Toxcore data file saved in the directory selected by
+  the user, with the file name `<profile_name>.tox`
 
-- ![](/badge/req.png) If the client supports user avatars, they will be stored
-  under the `avatars/` directory in the root of the client's working directory.
+- ![](/badge/rec.png) If the client supports user avatars, they should be stored
+  under the `avatars/` directory in the root of directory selected by the user.
 
 - ![](/badge/req.png) Avatars are named according to the following format:
   `PK.png`, where PK is the 64-character hexadecimal public key of its owner.
+  Additionally, aside from avatar data, hash of an avatarshould be stored in the
+  following format `PK.hash`.
```

Comment:
It should say `The client must check the file hash against the hash of the friend's currently saved avatar.`